go-runtime/modulecontext/module_context.go

```diff
@@ -2,59 +2,206 @@ package modulecontext
 
 import (
 	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"strconv"
 
-	cf "github.com/TBD54566975/ftl/common/configuration"
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+	_ "github.com/jackc/pgx/v5/stdlib" // SQL driver
 )
 
+type Ref struct {
```

Comment:
Can we use a `schema.RefKey` here? We're proliferating `Ref`s for no real reason I think, so I'm trying to consolidate them. `schema.Ref` has a method to retrieve a `RefKey`.

Reply:
Will do 👍